product-service/src/test/java/com/projectwork/product_service/ProductServiceApplicationTests.java

```diff
@@ -0,0 +1,7 @@
+package com.projectwork.product_service;
+
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class ProductServiceApplicationTests {
```

Comment:
this is incomplete. why there is no wiremock extension , testcontainers etc to run integration tests

Reply:
This is still incomplete i see... This is the integrationt test.